File: culqilib/src/main/java/com/culqilib/listener/TokenCallback.java
package com.culqilib.listener;

import com.culqilib.model.Error;

import org.json.JSONObject;

/**
 * Created by culqi on 2/7/17.
 */

public interface TokenCallback {

    void onSuccess(JSONObject token);

    void onError(Error error);

}

File: culqilib/src/main/java/com/culqilib/listener/TokenCallback.kt
package com.culqilib.listener


import com.culqilib.model.Token
import com.culqilib.model.TokenError

/**
 * Created by Louis Perdomo -> louis.perdomo79@gmail.com on 24/06/2019.
 */
interface TokenCallback {

    fun onSuccess(token: Token)

    fun onError(tokenError: TokenError)

}
